src/act.other.cpp

```diff
@@ -4592,7 +4592,7 @@ ACMD(do_syspoints) {
 
   // Require all arguments.
   if (!*arg || !*target || !*amt || !*reason || k <= 0 ) {
-    send_to_char(ch, "Syntax: syspoints <award|penalize> <player> <%samount> <Reason for award>.\r\n", k < 0 ? "POSITIVE " : "");
+    send_to_char(ch, "Syntax: syspoints <award|penalize> <player> <%samount> [for] <Reason for award>.\r\n", k < 0 ? "POSITIVE " : "");
```

Comment:
You've added a `[for]` here but not added logic to strip this out, which will cause `syspoints award player X for this reason` to output 'You have been awarded X system points for for this reason'.

Reply:
This is the syntax feedback message that's given when you type syspoints with incorrect/no args, not the actual message the logs, player, or staff member receives upon successful action.

It's also why the punctuation wasn't changed here.

It's there to remind staff that "for" gets inserted whether you include it or not (and I often created variations of that exact message you listed by accident multiple times).

I'm open to less awkward solutions or ideas.

Reply:
Oh, I see what you wanted to do here. Things in brackets like this traditionally mean optional arguments, so the meaning of this would be "you can write `for` here or not, whichever you prefer". Unless Epi / Vile have strong opinions, I think we should just revert this and make sure staff know that 'for' is automatically included.

Reply:
Epiphany has requested including for, sans brackets, and that is what I have done.